docs/quickstart.ipynb

```diff
@@ -199,14 +220,18 @@
     "\n",
     "The way every company chooses to expose a model will be different. In most cases models are integrated into other business applications and are rarely exposed directly to the Internet. By default, substratus will only serve the model within the Kubernetes cluster (with a Kubernetes [Service](https://kubernetes.io/docs/concepts/services-networking/service/) object). From here, it's up to you to expose the model to a wider network (e.g., the internal VPC network or the Internet) via annotated Service or Ingress objects.\n",
     "\n",
-    "In order to access the model for exploratory purposes, forward ports from within the cluster to your local machine."
+    "In order to access the model for exploratory purposes, forward ports from within the cluster to your local machine.\n"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "id": "ef93bee3-0c29-4435-a419-b7a21cf77ec7",
-   "metadata": {},
+   "metadata": {
+    "tags": [
+     "k port-forward server"
```

Comment:
should this use `kubectl` instead of `k`? not sure what tags does though

Reply:
nvm I get it now, this are arbitary tags we can use

Reply:
☝️  exactly. We might land on something more conventional (the literal command) but the tags are totally arbitrary